app/components/PalettePicker/index.js

```diff
@@ -47,9 +47,9 @@ class PalettePicker extends Component {
   // Set the original colors once then leave them alone
   _setOriginalColors(props) {
     if (
-      !this.state.originalColors.length
-      && props.options.color
-      && props.options.color.length
+      (!this.state.originalColors.length &&
```

Comment:
parans shouldn't be needed here but it's fine for readability, 

I normally do multiple line ifs like this:

```
if (!this.state.originalColors.length &&
  props.options.color &&
  props.options.color.length
) {
```